### ISSUE TITLE
[Bug]: Game of Skill creates NaN

### ISSUE THREAD

**Issue by levolz** [jasp-stats/jasp-issues#1662]
### JASP Version

0.16.1

### What analysis are you seeing the problem on?

Game of Skill

### What OS are you seeing the problem on?

Windows 11

### Bug Description

Game of Skill allows input (0, 0) for prior skill parameter & 0 input for points gained, leading to NaN output for the analytic solution

### Expected Behaviour

Not allow beta parameters = 0 as that is not defined

### Steps to Reproduce

Recreate with 0 input as described above

### Final Checklist

- [X] I have included a screenshot showcasing the issue, if any.
- [X] I have included a JASP file that causes the crash/bug, if any.
- [X] I have accurately described the bug, and steps to reproduce it.

**Comment by juliuspf** [comment]
hi @levolz. thanks for reporting. @Jiashun97 any ideas what to do here? Maybe input control or a footnote is required?

**Comment by Jiashun97** [comment]
> hi @levolz. thanks for reporting. @Jiashun97 any ideas what to do here? Maybe input control or a footnote is required?

You are right. The two parameters for beta distribution should be larger than 0. @vandenman Is it possible to add an input control for the table?

**Comment by vandenman** [comment]
@Jiashun97 I think you can set a lower bound with `minimum`.

**Comment by tomtomme** [comment]
@Jiashun97 
this bug is still valid with jasp 0.18.3. Are you still planning to fix it or should we reassign this issue?

**Comment by tomtomme** [comment]
still valid for latest 0.19 beta

**Comment by tomtomme** [comment]
@levolz 
Is this really something that needs to change?
It seems quite reasonable to me, that if you put in impossible or unplausible values that this results in NaN. Especially in an educational module you should alow the user to shoot themselfe in the foot  :D - right?

**Comment by levolz** [comment]
To be honest, four years in I don't have too much skin in the game anymore :D 
But if I recall correctly: conceptually, I found it a bit weird that essentially it is allowed to enter "impossible" prior parameter values (i.e., 0) as long as one enters points scored - while alpha & beta are defined to be >0, because you "directly" sample from the posterior beta-distribution & there are no explicit input checks, it gives out results except for the edge case described above.

If you think there is instructional value in this behaviour, go ahead not changing it ;)

PS: I just figured that while you can obviously enter fractional values for the prior parameters, when you also enter fractional parameters for the points scored it just loads forever until you change values. Maybe something to also address with input checks (or not, if it feels instructional :D)

**Comment by EJWagenmakers** [comment]
Certainly the input (points scored) should not be allowed to be fractional!
